package.json

```diff
@@ -22,7 +22,8 @@
     "build": "node build.js",
     "build:dev": "RUNTIME_API_HOST='https://zerobase-api-stage.herokuapp.com' RUNTIME_ENV=dev node build.js",
     "start:dev": "RUNTIME_ENV=dev node devServer.js",
-    "serve": "http-server ./public"
+    "serve": "http-server ./public",
+    "start": "node devServer.js"
```

Comment:
Hello friend, why did you add an npm start command here?

Reply:
Hi John, I'm on windows so the start:dev wasn't working for me. I created a regular start so I could run the site and make stylistic changes...could use some tips here.